homeworks/elena-savitskaya/homework-4/firstScript.js

```diff
@@ -0,0 +1,48 @@
+const array = [1, 50, 2, 5, 7, 10, 33, 22, 77];
+
+let moreTenArr = array.filter(function First(number) {
+  return number > 10;
+});
+
+alert(moreTenArr);
+console.log(array)
+
+
+function Second(num) {
```

Comment:
```suggestion
function second(num) {
```
should have lower case